shark/iree_utils.py

```diff
@@ -14,6 +14,7 @@
 
 import iree.runtime as ireert
 import iree.compiler as ireec
+from iree.compiler import tf as tfc
```

Comment:
Can we move this to iree_tf_utils.py ? We want to be able to run torch inference even if tensorflow and IREE tf is not installed.